datareservoirio/client.py

```diff
@@ -234,6 +239,28 @@ def delete(self, series_id):
         """
         return self._timeseries_api.delete(series_id)
 
+    @staticmethod
+    def timer(func):
```

Comment:
Rename the timer to `_timer` (with an underscore), so that it is hidden for the user